frame.go

```diff
@@ -202,6 +205,8 @@ const (
 	LocalQuorum Consistency = 0x06
 	EachQuorum  Consistency = 0x07
 	LocalOne    Consistency = 0x0A
+	Serial      Consistency = 0x08
+	LocalSerial Consistency = 0x09
```

Comment:
NIT: move `Serial` and `LocalSerial` above `LocalOne` to maintain logical order.